test/e2e_node/swap_test.go

```diff
@@ -58,6 +58,7 @@ var (
 
 var _ = SIGDescribe("Swap", "[LinuxOnly]", nodefeature.Swap, func() {
 	f := framework.NewDefaultFramework("swap-qos")
+	cleanupPods(f)
```

Comment:
let's move to under "WithSerial" context

Reply:
Done!